src/TEERegistry.sol

```diff
@@ -92,8 +93,18 @@ contract TEERegistry is Owned {
         returns (bool, bytes memory)
     {
         (bool success, bytes memory output) = AutomataDcapAttestationFee(attestationFeeContract).verifyAndAttestOnChain(quote);
+
         if (success) {
-            isVerified = true;
+            isVerified = true; // TODO: delete this once done testing
+
+            // since the verifyAndAttestOnChain call has succeeded, we can safely
+            // decode the output into the report body struct. We implicitly assume
```

Comment:
I guess we should check quote version and type, i.e. only TDX and v4 will succeed, everything else will revert

Reply:
good idea, I added a TODO and how to do it in a comment for a later PR